src/Quasar/Schema/TypeStats.purs

```diff
@@ -85,3 +108,131 @@ derive instance genericTypeStats ∷ Generic TypeStats
 
 instance showTypeStats ∷ Show TypeStats where
   show = gShow
+
+encode ∷ TypeStats → J.Json
+encode = CA.encode codec
+
+decode ∷ J.Json → Either String TypeStats
+decode = lmap CA.printJsonDecodeError <<< CA.decode codec
+
+codec ∷ CA.JsonCodec TypeStats
+codec = C.basicCodec dec enc
+  where
+  dec json = do
+    obj ← C.decode CA.jobject json
+    tag ← C.decode (CA.prop "kind" CA.string) obj
```

Comment:
Isn't this variant codec + `IsSymbol` instance?

Reply:
The variant codec looks for the `tag` property rather than `kind` which is needed for this schema.

That'd be an option worth having when defining codecs though :)

Reply:
Yeah, I mean that this codec does the same thing